Documentation/gettingstarted/istio.rst

```diff
@@ -205,16 +198,30 @@ Check the progress of the deployment (every service should have an
 
     $ watch "kubectl get deployments"
     NAME             DESIRED   CURRENT   UP-TO-DATE   AVAILABLE   AGE
-    details-v1       1         1         1            1           6m
-    productpage-v1   1         1         1            1           6m
-    reviews-v1       1         1         1            1           6m
+    details-v1       1         1         1            1           2m
+    productpage-v1   1         1         1            1           2m
+    reviews-v1       1         1         1            1           2m
+
+Once all the pods are available, verify that the application works by
+making a query from reviews pod to the productpage:
+
+::
+
+   $ kubectl exec -it $(kubectl get pod -l app=reviews -o jsonpath='{.items[0].metadata.name}') -c istio-proxy -- curl productpage:9080/productpage | grep "<title>.*</title>"
+
+Create an Istio ingress gateway for the productpage service:
```

Comment:
Why is the ordering different now for creating the Istio ingress gateway from before?

Reply:
This is the ordering used in the upstream example, and it makes sense to test the intra-service connectivity first, before adding in the external connectivity. Specifically, while doing this rebase I noticed that the prior method of getting the external URL did not work any more, and I needed to figure out where the problem was. Internal connectivity was fine, but had that not been fine,  would have wondered about the ingress gateway URL for nothing.